packages/viewer/src/interactions/interactionApi.ts

```diff
@@ -46,23 +46,36 @@ export class InteractionApi {
    */
   public async tap(
     position: Point.Point,
-    keyDetails: Partial<TapEventKeys> = {}
+    keyDetails: Partial<TapEventKeys> = {},
+    eventInfo: Partial<TapEventInfo> = {}
   ): Promise<void> {
-    this.emitTapEvent(this.tapEmitter.emit, position, keyDetails);
+    this.emitTapEvent(this.tapEmitter.emit, position, keyDetails, eventInfo);
   }
 
   public async doubleTap(
     position: Point.Point,
-    keyDetails: Partial<TapEventKeys> = {}
+    keyDetails: Partial<TapEventKeys> = {},
+    eventInfo: Partial<TapEventInfo> = {}
   ): Promise<void> {
-    this.emitTapEvent(this.doubleTapEmitter.emit, position, keyDetails);
+    this.emitTapEvent(
+      this.doubleTapEmitter.emit,
+      position,
+      keyDetails,
+      eventInfo
```

Comment:
What's the need for `eventInfo`? Shouldn't the event name already communicate that this is a double tap or long press?

Reply:
Yeah, I suppose in the case of a double tap this might not need to be populated. I can remove it